### PR TITLE
Run CI tests in parallel

### LINKED ISSUE #70: [cloudbuild] Fix parallel tests execution
Some kind of a race condition when testing buckets when running tests suites in parallel in cloud build.

```
× bucket-bindings: Test bucket bindings are correct
× Control Source Code Error /workspace/test/integration/authoritative/controls/authoritative.rb:97
767: unexpected token at ''
```

So far it looks like `gsutil` returns some wrong json bindings data for the buckets

### PR DESCRIPTION
**Pull request by cray0000**
- fix #70 

Up devtools to new image which fixes `gsutil` authorization issues.
Turn on parallel suites execution in cloud build.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.5.0
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.5.4
```
Comment (aaron-lane) [inline Makefile]: This can now be simplified:

```suggestion
DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0
```

File: build/int.cloudbuild.yaml [hunk 2]
```diff
@@ -236,4 +238,4 @@ tags:
 - 'integration'
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
-  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.5.0'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.5.4'
```
Comment (aaron-lane) [inline build/int.cloudbuild.yaml]:
```suggestion
_DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0'
```

File: build/lint.cloudbuild.yaml [hunk 3]
```diff
@@ -21,4 +21,4 @@ tags:
 - 'lint'
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
-  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.5.0'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.5.4'
```
Comment (aaron-lane) [inline build/lint.cloudbuild.yaml]:
```suggestion
 _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0'
```

**Comment by cray0000** [comment]
> Thanks @cray0000! Just some very simple changes requested.

Changed to use major version, thanks!

**Pull request merged**